clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/security/EcsCredentialsLifeCycleHandler.java

```diff
@@ -95,6 +96,14 @@ private void scheduleAgents(NetflixECSCredentials credentials) {
     Set<String> scheduledAccounts = ProviderUtils.getScheduledAccounts(ecsProvider);
     List<Agent> newAgents = new LinkedList<>();
     newAgents.add(new IamRoleCachingAgent(credentials, amazonClientProvider, iamPolicyReader));
+    newAgents.add(
+        new ApplicationCachingAgent(
+            credentials,
+            "us-east-1",
```

Comment:
I am not super familiar with the caching agents. May I know why the region is hard coded over here?

Reply:
We only want one of these caching agents. The region doesn't really matter in this case, but to extend the abstract caching agent, you need to define a region.